app-i18n/mozc/mozc-2.23.2785.102.recipe

```diff
@@ -2,12 +2,12 @@ SUMMARY="Japanese input method editor"
 DESCRIPTION="A Japanese Input Method Editor (IME) designed for multi-platform. \
 This is open source version."
 HOMEPAGE="https://github.com/google/mozc/"
-COPYRIGHT="2010-2016, Google Inc."
+COPYRIGHT="2010-2018, Google Inc."
 LICENSE="BSD (3-clause)"
-REVISION="2"
-fileVersion="v2.20.2677.102.6"
+REVISION="1"
+fileVersion="v2.23.2785.102.1"
 SOURCE_URI="https://github.com/hanya/mozc/releases/download/$fileVersion/mozc-$fileVersion.tar.bz2"
-CHECKSUM_SHA256="cdb43a88d82db185299a8e396fccfe76ddbbe5a3c6b219220a36b55115cff4ac"
+CHECKSUM_SHA256="17ce4ee5d6a55678d804890d7e2fa7ffd7f852a9673e3b79671613c9167469f5"
 SOURCE_DIR="mozc"
 if [ $effectiveTargetArchitecture == "x86" ]; then
```

Comment:
```diff
-if [ $effectiveTargetArchitecture == "x86" ]; then
+if [ "$effectiveTargetArchitecture" = x86 ]; then
```

(The double quotes are unneded around the `x86`, and very welcome around the `$effectiveTargetArchitecture`.)
(`=` works with both `/bin/sh` and `/bin/bash`, whreas `==` only works with `/bin/bash`.)